scripts/qa_crowbarsetup.sh

```diff
@@ -4156,6 +4156,13 @@ function onadmin_testsetup
             safely pushd qa-automation
         fi
 
+        # lets see if we can ping the cephmons_names network
+        ping ${cephmons_names[0]} -c 3
+        if [[ $? != 0 ]]; then
+            # Add crowbar to the $net_name network so $net_name.<host> can route.
+            add_node_to_network $(hostname -f) $net_name
```

Comment:
This will not do the right thing with our physical clouds that have crowbar only bridged to the admin vlan... might be better to run the tests on an existing ceph node?

Reply:
Agreed, please try to run the tests from another node with the needed access instead of manipulating the network in a way that does not reflect our recommendations for customers.